ports/geckolib/glue.rs

```diff
@@ -308,15 +310,16 @@ pub extern "C" fn Servo_GetComputedValuesForPseudoElement(parent_style: *mut Ser
                                                           pseudo_tag: *mut nsIAtom,
                                                           raw_data: *mut RawServoStyleSet,
                                                           is_probe: bool)
-     -> *mut ServoComputedValues {
+     -> ServoComputedValuesStrong {
     debug_assert!(!match_element.is_null());
 
     let parent_or_null = || {
         if is_probe {
-            ptr::null_mut()
+            ComputedValues::null_strong()
         } else {
             Servo_AddRefComputedValues(parent_style);
-            parent_style
+            // XXXManishearth temporary till we can borrow
+            unsafe {transmute(parent_style)}
```

Comment:
I think you can do this more nicely now with your new helpers.
ComputedValues::with(|cv| { ComputedValues::from_arc(cv.clone()) })